packages/raccoon-city-ui/src/components/Main/ChessGrid/FlatSidebarInfo/FlatSidebarInfo.tsx

```diff
@@ -136,6 +140,10 @@ export function FlatSidebarInfo(props: FlatSidebarInfoProps) {
     };
 
     const flat = data.getFlatSidebarInfo;
+    const isHasVR: boolean = Boolean(flat?.layout?.images?.VR?.length);
```

Comment:
you don't need use type explicitly because you've already provided the default value